h2o-py/h2o/model/model_base.py

```diff
@@ -390,6 +390,20 @@ def scoring_history(self):
         print("No score history for this model")
 
 
+    def ntrees_actual(self):
+        """
+        Returns actual number of trees in GBM model. If early stopping enabled, GBM can reset the ntrees value.
```

Comment:
mentions GBM when in facts applies to other algos

Reply:
fixed